package.json

```diff
@@ -87,7 +91,7 @@
     "eslint-loader": "^1.6.3",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^3.0.2",
-    "eslint-plugin-react": "^6.9.0",
+    "eslint-plugin-react": "6.9.0",
```

Comment:
This was the source of the mystery build errors: `eslint-plugin-react` v6.10 introduced a breaking change, and there's some discussion about it in the repo (see yannickcr/eslint-plugin-react#1117). Pinning to 6.9.0 directly will maintain the expected behavior until that issue gets sorted.